integration-test/common/conftest.py

```diff
@@ -1,8 +1,41 @@
+""" This contains pytest hook implementations.
```

Comment:
nit: This top line comment doesn't say what the hooks are *for*. I'd suggest something like "implements pytest hooks to run MachSuite tests on Aladdin".

Reply:
Changed the top line commented.